src/ConnectedMode/ConnectedModePackage.cs

```diff
@@ -84,17 +84,24 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
             hotspotStoreMonitor = componentModel.GetService<ILocalHotspotStoreMonitor>();
             await hotspotStoreMonitor.InitializeAsync();
 
-            // Trigger an initial update of suppressions (These classes might have missed the initial solution binding
-            // event from the ActiveSolutionBoundTracker)
-            // See https://github.com/SonarSource/sonarlint-visualstudio/issues/3886
+            logger.WriteLine(Resources.Package_Initialized);
+        }
+
+        /// <summary>
+        /// Trigger an initial update of suppressions (These classes might have missed the initial solution binding
+        /// event from the ActiveSolutionBoundTracker)
+        /// See https://github.com/SonarSource/sonarlint-visualstudio/issues/3886
+        /// </summary>
+        private void LoadServicesAndDoInitialUpdates(IComponentModel componentModel)
```

Comment:
With how many there are now I thought it made sense to put it in a seperate method.

Reply:
The doc comment on the method needs to updated.